_guides/responsive_amp.md

```diff
@@ -5,17 +5,124 @@ order: 0
 folder: responsive
 ---
 
-AMP balances its support for developers and users.
-It helps developers create content that looks and behaves as designed,
-while protecting the user experience from slow page loads
-and juttery interactions.
-
-In AMP, only inline styles are allowed; this removes extra HTTP requests
-from the critical rendering path.
-AMP uncouples document layout from resource loading;
-it requires all external resources to state their size
+It is super easy to make responsive elements in AMP.
+Just put `layout=responsive` on them.
+
+{% include toc.html %}
+
+## Create responsive images
+
+All externally-loaded resources including images,
+must have a specified size and position
+so as the resources load, the page won't jump and reflow.
+
+Create responsive images
+by specifying the width and height
+and setting layout to responsive:
+
+{% highlight html %}
+<amp-img
+    src="/img/amp.jpg"
+    width="1698"
+    height="2911"
+    layout="responsive"
+    alt="an image">
+</amp-img>
+{% endhighlight %}
+
+This `amp-img` element automatically fits the width
+of its container element,
+and its height is automatically set to the aspect ratio
+determined by the given width and height:
+
+<amp-img src="/docs/assets/responsive_amp_img.png" width="500" height="857" layout="responsive"></amp-img>
+
+See also [AMP by Example's amp-img](https://ampbyexample.com/components/amp-img/).
```

Comment:
Maybe here recommend to always use srcset. I'd lean towards using srcset in the examples.